spec/rubocop/cli/cli_autocorrect_spec.rb

```diff
@@ -945,6 +945,46 @@ class Bar
       RUBY
   end
 
+  it 'corrects multiline array brace without crashing' do
```

Comment:
Why didn't you add those tests to the spec of the cop instead? It seems to me this is the natural place for them.

Reply:
Done.